docs/_includes/navbar.html

```diff
@@ -24,15 +24,15 @@
         We might be on an externally hosted documentation site.
         Please keep the site.FLINK_WEBSITE_URL below to ensure a link back to the Flink website.
         {% endcomment %}
-	<a href="{{ site.FLINK_WEBSITE_URL }}index.html" title="Home">
+	<a href="{{ site.FLINK_WEBSITE_URL }}/index.html" title="Home">
 	  <img class="hidden-xs hidden-sm img-responsive"
-	       src="{{ site.baseurl }}img/logo.png" alt="Apache Flink Logo">
+	       src="{{ site.baseurl }}/img/logo.png" alt="Apache Flink Logo">
 	</a>	
 	<div class="row visible-xs">
 	  <div class="col-xs-3">
-	    <a href="{{ site.baseurl }}index.html" title="Home">  
+	    <a href="{{ site.baseurl }}/index.html" title="Home">
```

Comment:
This will point the Home button to http://ci.apache.org instead of the documentation. {{ site.baseurl }} is actually not set in the config. We could set it to the current docs or just keep the relative links which worked fine.